src/main/java/databaseInteracting/Client.java

```diff
@@ -0,0 +1,163 @@
+package databaseInteracting;
+
+import generated.addProductRequest;
+import generated.addProductResponse;
+import generated.addProductsToShoppingCartRequest;
+import generated.addProductsToShoppingCartResponse;
+import generated.calculateTotalAmountRequest;
+import generated.calculateTotalAmountResponse;
+import generated.createAShoppingCartRequest;
+import generated.createAShoppingCartResponse;
+import generated.listByIdRequest;
+import generated.listByIdResponse;
+import generated.listProductRequest;
+import generated.listProductResponse;
+import generated.listShoppingCartProductsRequest;
+import generated.listShoppingCartProductsResponse;
+import generated.productGrpc;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+
+import java.util.Iterator;
+import java.util.Scanner;
+
+public class Client {
+  /**
+   * Main Method of class.
+   * Class with a menu to interact with the database and the server side.
+   *
+   * @param args Array of String to pass parameters.
+   */
+  public static void main(String[] args) {
+    String channelName = args[0];
+    int portNumber = Integer.parseInt(args[1]);
+    ManagedChannel channel = ManagedChannelBuilder
+        .forAddress(channelName, portNumber).usePlaintext().build();
+    productGrpc.productBlockingStub prodStub = productGrpc.newBlockingStub(channel);
+    Scanner sc = new Scanner(System.in);
+    int option;
+    int aux;
+    do {
+      System.out.println("Select one option:");
+      System.out.println("[1] - List all products");
+      System.out.println("[2] - Search a product");
+      System.out.println("[3] - Add a product");
+      System.out.println("[4] - Create a Shopping Cart");
+      System.out.println("[5] - Add a product to Shopping Cart");
+      System.out.println("[6] - Show the Shopping Cart");
+      System.out.println("[7] - Calculate the total purchase amount.");
+      System.out.println("[8] - Import products from a file.");
+      System.out.println("[9] - Export products to a file.");
+      System.out.println("[10] - Finish execution");
+      option = sc.nextInt();
+      if (option == 1) {
+        try {
+          listProductRequest request = listProductRequest.newBuilder().build();
+          Iterator<listProductResponse> listProductResponseIterator = prodStub.listProduct(request);
+          while (listProductResponseIterator.hasNext()) {
+            listProductResponse next = listProductResponseIterator.next();
+            System.out.printf("Id: %d Name: %s Stock: %d Price: %.2f\n",
+                next.getId(), next.getName(), next.getStock(), next.getPrice());
+          }
+        } catch (Exception e) {
+          e.printStackTrace();
```

Comment:
Not a good practice to print the stack trace

Reply:
done.